chunks.go

```diff
@@ -91,6 +93,8 @@ func newChunkWriter(dir string) (*chunkWriter, error) {
 	return cw, nil
 }
 
+// XXX: should this be called head instead of tail? We also call the
+// currently active block the "head" instead of the "tail".
```

Comment:
Yea, probably makes sense. Or rename current heads to tails?
I cannot see an immediate wrong/right way for this. I kinda went with head based on what the 1.x storage has.